templates/apollo-runtime-ts/README.md

```diff
@@ -0,0 +1,44 @@
+## apollo-starter-ts
+
+Template that showcases Graphback runtime generation capabilities using 
+resolver layer generated in memory. 
+
+### Example 
+
+To use runtime capabilities developers will need to create data provider.
+Currently 2 data providers are supported:
+
+ - KnexDBDataProvider
+ - PgKnexDBDataProvider
+ - RestDataProvider (Looking for contributions)
+
+Then developers can create runtime instance:
+
+```ts
+    const backend = new GraphQLBackendCreator(`your model string here`, {});
+    const dbClientProvider = new PgKnexDBDataProvider(client);
```

Comment:
It's not clear here what `client` is, or how to define it. An example should be include above this maybe?

Reply:
Done